tests/test_ag3.py

```diff
@@ -1829,6 +1829,85 @@ def test_h1x_gwss():
     assert np.all(h1x >= 0)
 
 
+def test_average_fst():
+    ag3 = setup_ag3()
+    region = "3L:15,000,000-16,000,000"
+    cohort1_query = "cohort_admin2_year == 'ML-2_Kati_colu_2014'"
+    cohort2_query = "cohort_admin2_year == 'ML-2_Kati_gamb_2014'"
+    n_jack = 200
+    site_mask = "gamb_colu"
+
+    fst_hudson, se_hudson = ag3.average_fst(
+        region=region,
+        cohort1_query=cohort1_query,
+        cohort2_query=cohort2_query,
+        n_jack=n_jack,
+        site_mask=site_mask,
+    )
+
+    # check data
+    assert isinstance(fst_hudson, np.float64)
+    assert isinstance(se_hudson, np.float64)
+
+    # check data
+    assert np.isscalar(fst_hudson)
+    assert np.isscalar(se_hudson)
+
+    # check some values
+    assert np.allclose(fst_hudson, 0.039983, rtol=1e5), fst_hudson
+    assert np.allclose(se_hudson, 0.003327, rtol=1e5), se_hudson
+
+
+def test_pairwise_average_fst():
+    ag3 = setup_ag3()
+    region = "3L:15,000,000-16,000,000"
+    cohorts = "cohort_admin1_year"
+    sample_query = "country == 'Mali' and taxon == 'gambiae'"
+    n_jack = 200
+    site_mask = "gamb_colu"
+
+    test_df = pd.DataFrame(
+        {
+            "cohort1": [
+                "ML-2_gamb_2004",
+                "ML-2_gamb_2004",
+                "ML-2_gamb_2014",
+            ],
+            "cohort2": [
+                "ML-2_gamb_2014",
+                "ML-3_gamb_2012",
+                "ML-3_gamb_2012",
+            ],
+            "fst": [
+                0.037249514094550934,
+                0.04154191785684654,
+                0.0,
+            ],
+            "se": [
+                0.0020406887352541958,
+                0.002369740033208285,
+                0.0008138514674580574,
+            ],
+        }
+    )
+
+    fst_df = ag3.pairwise_average_fst(
+        region=region,
+        cohorts=cohorts,
+        sample_query=sample_query,
+        n_jack=n_jack,
+        site_mask=site_mask,
+    )
+
+    # check data
+    assert isinstance(fst_df, pd.core.frame.DataFrame)
+
+    # check some values
+    pd.testing.assert_frame_equal(fst_df, test_df, rtol=1e5)
+    assert np.all(fst_df["fst"] <= 1)
+    assert np.all(fst_df["fst"] >= -0.05)
```

Comment:
```suggestion
    assert np.all(fst_df["fst"] >= 0)
```

Because negative values are being clipped to zero, we could check that here.